README.md

```diff
@@ -1,4 +1,25 @@
 # File Checker
 [![Build Status](https://travis-ci.org/iattempt/FileChecker.svg?branch=master)](https://travis-ci.org/iattempt/FileChecker)
 
-A checker/detector could check name of file/directory is expectable or not.
\ No newline at end of file
+A checker/detector could check name of file/directory is expectable or not.
+
+# Features
++ Only test particular files/directories.
++ Ignore particular files/directories.
++ Supports checking of most common format of letter-cases (upper, lower, camel, and pascal-case).
++ Supports checking of convention of word separators (underscore-to-dash/dash-to-underscore).
+
+# How does this work?
```

Comment:
I would add another section for building/running.

Reply:
Ok, will you create other branch for this or in this PR?

Reply:
No, I meant to ask you to do it, since you are updating the documentation. It should be a fairly small section, and very easy to add. 😁

Reply:
@arnavb sorry for that I cannot understood the "No, I meant to ask you to do it." sentence, will you work on this or do you want me to do this😅?

Reply:
It's ok @iattempt. I am asking _you_ do to it (unless you _want_ me to work on it). This is easier, since you have already started documenting everything!

Reply:
@arnavb I have some work to do, could you help for this if you have time. thanks~

Reply:
Sure, I'll do it!